ansible/roles/linux-common/tasks/main.yml

```diff
@@ -26,3 +25,52 @@
 # Enables volumes to be referred to by tag for setting up mounts in cloud-config
 - include_role:
     name: linux-volumes-by-tag
+
+# Use linux-generic to track linux-modules-extra
+- name: Install InfiniBand support (deb based)
+  ansible.builtin.apt:
+    name: linux-generic
+    state: latest
+    update_cache: yes
+  when: ansible_os_family == "Debian"
+  become: true
```

Comment:
All these `become`s are not necessary, because the playbooks specify `become: yes` so all tasks are executed with root privileges unless otherwise configured on the task.